src/util/constants.ts

```diff
@@ -63,8 +63,21 @@ export enum ExitCodes {
   Failed = 9,
   OutOfSpace = 3,
   BrowserCrashed = 10,
-  InterruptedGraceful = 11,
-  InterruptedImmediate = 13,
+  Cancelled = 11,
+  FailedLimit = 12,
+  Interrupted = 13,
+  SizeLimit = 14,
+  TimeLimit = 15,
+  DiskUtilization = 16,
```

Comment:
I think this may be a duplicate of exit code 3 (OutOfSpace)

Reply:
I think they're slightly different -- one is projected DiskUtilization based on the --diskUtilization, the other is crawler has actually run out of space already.

Reply:
Ah, good point